README.md

```diff
@@ -59,3 +59,23 @@ This can be enabled with the `orangepi` feature:
 verson = "0.2"
 features = ["orangepi"]
 ```
+
+##Development Mode
+
+In development mode, `rust-wiringpi` is compiled as rust-native library excluding original WiringPi.
+And binding functions are replaced by dummy functions that output simple logs to stdout.
+With this mode, you can build and debug your project on wiringpi-not-supported platform such as macOS.
```

Comment:
Just a bit of grammar touch-up and rephrasing suggestion:

```
In development mode, `rust-wiringpi` is compiled as a rust-native library excluding the original WiringPi.
And binding functions are replaced by dummy functions that output simple logs to stdout.
With this mode, you can build and debug your project on platforms that does not support WiringPi.
```

I don't think there is any reason to mention any specific OS :)